.github/workflows/docker-build-only.yml

```diff
@@ -87,28 +87,27 @@ jobs:
 
       - name: Configure image variables
         run: |
-          server_sha_short=${GITHUB_SHA:0:7}
-          echo "IMAGE_TAG=sha-$(git submodule status -- temporal | cut -c2-8)" >> $GITHUB_ENV
+          echo "IMAGE_SHA_TAG=sha-$(git submodule status -- temporal | cut -c2-8)" >> $GITHUB_ENV
           echo "TAG_LATEST=true" >> $GITHUB_ENV
 
       - name: Bake images
         run: BAKE_OUTPUT=docker make build-native
```

Comment:
is it a problem that IMAGE_BRANCH_TAG isn't set here? the Makefile will guess it?

Reply:
Right; the Makefile will assign the default of `git rev-parse --abbrev-ref HEAD`. Which is not a problem here since this test never pushes anything anyway.

Reply:
ah, I didn't realize this never pushed